<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Путь в SVN:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string> (например, https://svn.example.com/trunk/)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="md5Edit"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Имя файла: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string> (можно оставить пустым, если ищете только по хэшу)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="filenameEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>md5 хэш:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string> (можно оставить пустым, если ищете только по имени)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pathEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Поиск</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/free-icon-search-4024513.png</normaloff>../../../Downloads/free-icon-search-4024513.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="resultEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="detectMd5Button">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Узнать md5 хэш</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/free-icon-md5-1792205.png</normaloff>../../../Downloads/free-icon-md5-1792205.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="button_close">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Закрыть </string>
      </property>
      <property name="icon">
       <iconset theme="contact-new">
        <normaloff>.</normaloff>.</iconset>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1360</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
